lib/fhir_models/fhir_ext/structure_definition.rb

```diff
@@ -426,7 +426,7 @@ def valid_json?(json)
           else
             element.max.to_i
           end
-        if (nodes.size < min) && (nodes.size > max)
+        if (nodes.size < min) || (nodes.size > max)
```

Comment:
I prefer the boolean logic. It is easier to read for programmers of all backgrounds.